Bottom: 4a6d3d785bbd45aee4fba5eed591a50d44e92235
Top:    2e7faf5089bf9a65452b3d541b34f01b581ec67d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 15:13:55 +0000

Refresh of extend-object3d-code-to

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index a3e4b83..cfce41a 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -255,8 +255,6 @@ get_contour_edge_n_is_round (PLINE *contour, int n)
     }
 #endif
 
-  return false;
-
   while (n > 0) {
     edge = edge->next; /* The VNODE structure is circularly linked, so wrapping is OK */
     n--;
@@ -1540,7 +1538,7 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
     END_LOOP;
 #endif
 
-#if 0
+#if 1
     fprintf (stderr, "Accumulating pin + via pads\n");
     r_search (PCB->Data->pin_tree, &bounds, NULL, pv_copper_callback, &info);
     r_search (PCB->Data->via_tree, &bounds, NULL, pv_copper_callback, &info);
@@ -1599,7 +1597,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
                               copper_appearance,
                               NULL));
 
-
     //break;
   }
